rpc/js/.eslintrc.cjs

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+module.exports = {
+  root: true,
+  extends: ['@viamrobotics/eslint-config'],
+  parserOptions: {
+    project: ['./tsconfig.json'],
+  },
```

Comment:
Since we're in a large, multilevel repo with a few package.json files, we've found things to be a little more consistent if we include the `tsconfigRootDir: __dirname` setting

```suggestion
  parserOptions: {
    tsconfigRootDir: __dirname,
    project: './tsconfig.json',
  },
```

Without it, ESLint can get a little confused about where to find the TSConfig